projects/Apps/editionsSrc/src/helpers/__tests__/files.spec.ts

```diff
@@ -1,10 +1,10 @@
 import MockDate from 'mockdate'
 import {
-    matchSummmaryToKey,
+    // matchSummmaryToKey,
     downloadAndUnzipIssue,
     issuesToDelete,
 } from '../files'
-import { issueSummaries } from '../../../../Apps/common/src/__tests__/fixtures/IssueSummary'
+// import { issueSummaries } from '../../../../Apps/common/src/__tests__/fixtures/IssueSummary'
```

Comment:
Should this be deleted or uncommented?

Reply:
Its a fixable issue as we need to refactor the tests to be able to run the same ones and pass the typescript linting